### PR TITLE
License Code and Knowledge base URL

### LINKED ISSUE #2658: Issue in get license key in email.
**Issue:-**
Unable to get the license key in email, in purchase confirmation email. unable to click the click here option.
image

### PR DESCRIPTION
**Pull request by Gurmeenkour99**
Corrected the redirection for knowledge base and get license Code URL in the purchase email template and also added the option in system settings to add the knowledge base URL.
Fixes #2658 #2751 #2752 

Note : Need to run php artisan migrate and php artisan db:seed --class=Database\\Seeders\\v3_0_5\\DatabaseSeeder

### CODE REVIEW

**Review by sowmiyasubramaniyan — DISMISSED** [review]

**Review by sandesh556 — APPROVED** [review]

**Comment by Sabaric12** [comment]
Tested this Prrwith the existing instance
Issue 1:
In the Store, the Products are not showing properly it is overlapping with each other
image
Issue 2:
After purchasing the product email is not receiving
Total time in days: 5 hours
Total issues: 2
Minor - 0 
Major - 0 
Critical - 2
No of times PR was sent for Functionality correction:1
PR breaks existing functionality: No

**Comment by Gurmeenkour99** [comment]
@Sabaric12 The overlapping issue you have mentioned is an existing issue and this has been resolved now in client panel theme PR and the email sending issue is due to database setup or some invalid details in records. 
i also tried this and i'm receiving the mail .

image

**Comment by Sabaric12** [comment]
Round 1: Testing Summary
Tested this PR with the existing instance in the server
The mentioned issues are working fine now
Before Fix:
image
After Fix:
Screenshot from 2023-12-12 11-13-10
Total time in days: 1 day
Total issues: 0
Minor - 0 
Major - 0 
Critical - 0
No of times PR was sent for Functionality correction:0
PR breaks existing functionality: No
Note:
The overlap issue in the Client panel purchase product is fixed in the Client panel theme PR 
https://github.com/ladybirdweb/agora-invoicing-community/pull/2787
image

**Review by Sabaric12 — DISMISSED** [review]

**Review by ganthimathin — CHANGES_REQUESTED** [review]
Tested the issues with existing DB.

License code page is showing now. But Knowledge base URL- is not getting launched once clicked from purchase confirmation mail.

**Comment by Gurmeenkour99** [comment]
@ganthimathin please try to run seeder command php artisan db:seed --class=Database\\Seeders\\v3_0_5\\DatabaseSeeder

**Review by ganthimathin — DISMISSED** [review]
Verified this issue in server instance. There its working fine.
Both licencode and knowledge base URL is getting opened.

**Comment by ganthimathin** [comment]
Total time in days: 5 hours
Total issues: 0
Minor - 0
Major - 0
Critical - 0
No of times PR was sent for Functionality correction:0
PR breaks existing functionality: No

**Comment by ganthimathin** [comment]
> @ganthimathin please try to run seeder command php artisan db:seed --class=Database\Seeders\v3_0_5\DatabaseSeeder

I ran the command. But issue with my local host only. Now in server instance its working fine..

File: database/seeders/v3_0_5/DatabaseSeeder.php [hunk 1]
```diff
@@ -15,6 +17,85 @@ class DatabaseSeeder extends Seeder
     public function run()
     {
         $this->call(ExpiryMailDaySeeder::class);
+                
+    Template::where('id',4)->update(['name' => 'Purchase confirmation', 'type' => 7, 'url' => 'null', 'data' => '<table style="background: #f2f2f2; width: 700px;" border="0" cellspacing="0" cellpadding="0">
```
Comment (sandesh556) [inline database/seeders/v3_0_5/DatabaseSeeder.php:21]: How do you know the id is going to be 4?

**Review by sandesh556 — CHANGES_REQUESTED** [review]

**Pull request merged**